packages/gatsby/src/utils/get-latest-apis.js

```diff
@@ -0,0 +1,33 @@
+const path = require(`path`)
+const fs = require(`fs-extra`)
+const axios = require(`axios`)
+
+const API_FILE = `https://unpkg.com/gatsby/apis.json`
+const ROOT = path.join(__dirname, `..`, `..`)
+const OUTPUT_FILE = path.join(ROOT, `latest-apis.json`)
+
+module.exports = async function getLatestAPI() {
+  /*
+   * Happy path `postinstall` script created the file
+   */
+  if (await fs.exists(OUTPUT_FILE)) {
+    return fs.readJSON(OUTPUT_FILE)
+  }
```

Comment:
I think trying to load from file makes more sense as fallback and to prefer fetching fresh if possible.

In case of creating new gatsby project - cached `latest-apis.json` will be equaivalent to what `gatsby` package was doing already - as it will be list of APIs that is only up-to-date with their version of gatsby. If later on we add new API and implement it in plugin. Users install plugin - he still will have cached `latest-apis.json` that have no data about this new API

Reply:
Yeah good call! Will fix.

Reply:
Fixed!